app/javascript/components/pages/TripShow.js

```diff
@@ -17,9 +17,17 @@ class TripShow extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      success: false
+      success: false,
+      isDeleted: false
     }
   }
+
+handleClick = () => {
+  this.props.deleteTrip(this.props.trip.id)
+  this.setState({ isDeleted: true})
+  console.log("Handle Click")
```

Comment:
Aside from setting state here, does isDeleted do anything specific on this page?  Also, please remove this console log.